File: csv/ringo/delete/csv_delete.js
#! /usr/bin/ringo
// ------------------------------------------------------------------
//
//	csv_update.js
//
//						May/10/2011
//
// ------------------------------------------------------------------
var text_manipulate = require ('/var/www/data_base/common/ringo_common/text_manipulate.js');
// ------------------------------------------------------------------
print	("*** 開始 ***");
var args = require('system').args;
var file_txt = args[1];
var id_in = args[2];
print	(id_in);

var dict_aa = text_manipulate.csv_read_proc (file_txt);

delete dict_aa[id_in];

text_manipulate.dict_display_proc (dict_aa);
text_manipulate.csv_write_proc (file_txt,dict_aa);

print	("*** 終了 ***");
// ------------------------------------------------------------------

File: csv/jruby/delete/csv_delete.rb
#! /usr/bin/jruby
# -*- coding: utf-8 -*-
#
#	delete/csv_delete.rb
#
#				Jul/02/2011
#
# -------------------------------------------------------------
load '/var/www/data_base/common/ruby_common/text_manipulate.rb'
#
# -------------------------------------------------------------
puts "*** 開始 ***"
#
file_in = ARGV[0]
#
id_in = ARGV[1]
#
puts id_in
#
dict_aa = csv_read_proc(file_in)
#
dict_aa = dict_delete_proc(dict_aa,id_in)
dict_display_proc(dict_aa)
csv_write_proc(file_in,dict_aa)
#
puts "*** 終了 ***"
#
# -------------------------------------------------------------
